components/NewsCard/NewsCard.js

```diff
@@ -0,0 +1,44 @@
+import Image from "next/image";
+import classNames from "classnames";
+import styles from "./NewsCard.module.scss";
+import {
+  colorAccentPurple,
+  colorAccentBlue,
+  colorAccentGreen,
+  colorAccentOrange,
+  colorAccentYellow,
+  colorAccentRed,
+} from "../../styles/exportColorVars.module.scss";
+import img from "../../public/img/court-placeholder.jpg";
+
+export default function NewsCard() {
+  function pickColor() {
+    const colorsArr = [
+      colorAccentPurple,
+      colorAccentBlue,
+      colorAccentGreen,
+      colorAccentOrange,
+      colorAccentYellow,
+      colorAccentRed,
+    ];
+    const randomColor = colorsArr[Math.floor(Math.random() * colorsArr.length)];
+    return randomColor;
```

Comment:
NIT: Возможно так было б более читабельно. Иначе объявление переменной прямо перед её возвратом в большинстве случаев не целесообразно.

```suggestion
    const randomColorIndex = Math.floor(Math.random() * colorsArr.length);
    return colorsArr[randomColorIndex];
```